services/mailer/mail_issue.go

```diff
@@ -41,7 +41,7 @@ func mailIssueCommentToParticipants(ctx *mailCommentContext, mentions []int64) e
 		return fmt.Errorf("LoadPullRequest(): %v", err)
 	}
 
-	// Enough room to avoid reallocations
+	// Enough room to avoid reallocation's
```

Comment:
```suggestion
	// Enough room to avoid reallocations
```
reallocations is correct